config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -695,9 +706,10 @@ periodics:
           memory: 6Gi
   annotations:
     testgrid-dashboards: sig-node-containerd
-    testgrid-tab-name: image-validation-cos-e2e
-- interval: 1h
-  name: ci-cos-containerd-e2e-ubuntu-gce
+    testgrid-tab-name: ci-containerd-cos-image-validation-e2e-gce
+
+- name: ci-containerd-cgroupTODO-ubuntu-image-validation-e2e-gce-release-main
```

Comment:
remove "TODO"

Reply:
need to check the cgroup used, os used from this tab probably?
https://testgrid.k8s.io/sig-node-release-blocking#ci-crio-cgroupv1-node-e2e-conformance